src/main/java/org/jenkinsci/plugins/docker/workflow/client/WindowsDockerClient.java

```diff
@@ -61,6 +73,9 @@ public String run(@Nonnull EnvVars launchEnv, @Nonnull String image, @CheckForNu
 
     @Override
     public List<String> listProcess(@Nonnull EnvVars launchEnv, @Nonnull String containerId) throws IOException, InterruptedException {
+        if (isContainerUnix) {
+            return listProcessUnixContainer(launchEnv, containerId);
```

Comment:
Why not supper.listProcesses(...) ?  DockerClient.listProcesses is already unix specific code.  And include "-eo", "pid,comm" to remove the {} that busybox can put around the process names.